src/nvme/tree.c

```diff
@@ -971,9 +972,7 @@ static void __nvme_free_ctrl(nvme_ctrl_t c)
 {
 	struct nvme_path *p, *_p;
 	struct nvme_ns *n, *_n;
-
```

Comment:
Please avoid changing white space changes here (btw, this new line here is okay)